portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -0,0 +1,50 @@
+package com.google.sps.servlets;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Date;
+
+import com.google.sps.data.Comment;
+import com.google.sps.data.CommentBuilder;
+
+import com.google.gson.Gson;
+
+@WebServlet("/comments")
+public final class CommentsServlet extends HttpServlet {
+    private List <Comment> getCommentsList() {
+        ArrayList <Comment> commentsList = new ArrayList <Comment>();
+
+        commentsList.add(new CommentBuilder()
+            .setId(1)
+            .setName("Max B")
+            .setDescription("My prev comment!")
+            .setPostDate(new Date(1595318478000L))
+            .build()
+        );
+
+        commentsList.add(new Comment(1, "Max B", "Hello y'all!", new Date(1595320478000L)));
+        commentsList.add(new Comment(2, "Maksim S", "Hey folks!", new Date(1595322478000L)));
+        commentsList.add(new Comment(3, "Remus N", "Hey Everyone!", new Date(1595324478000L)));
+        commentsList.add(new Comment(4, "Nick T", "Welcome everyone! Have fun! :)", new Date(1595326478000L)));
+
+        return commentsList;
+    }
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
I need to double check, bit I believe best practice is to not throw exceptions into users face.
a) It's not user friendly.
b) It might expose sensitive application details to a malicious user.